.github/workflows/test-python-client.yaml

```diff
@@ -0,0 +1,61 @@
+name: Test Raccoon Python Client
+on:
+  push:
+    paths:
+      - "clients/python/**"
+    branches:
+      - main
+  pull_request:
+    paths:
+      - "clients/python/**"
```

Comment:
We are running on tests on main, on push and for pull request that have changes in the path clients/python/** for any PR

Reply:
I meant can we add branch main for pull requests also?

Reply:
Won't that mean any pull_request which doesn't target the main branch (ex: current MR) won't have the tests running. CMIIW

Reply:
yes, so either we do that, or we can run tests on every push also

Reply:
Running on every push which hasn't raised pull request also doesn't make sense, since in that case it doesn't mean there's any plan to merge it.

Reply:
but then why even on PRs which are not on main?

Reply:
An example would be current PR. In case, there is a feature branch and multiple sub feature branches. If we target only on main, these pipelines won't run.

Reply:
Resolving it